docs/latest/dev/releasing.md

```diff
@@ -24,7 +24,7 @@ export GITHUB_REMOTE=origin
 ```
 
 1. Clone the repo, checkout the `main` branch, ensure it’s up-to-date, and your local branch is clean.
-2. Create a topic branch for adding the release notes. Refer to previous [release notes][] for additional details.
+2. Create a topic branch for adding the release notes and updating the [VERSION][] file with the release version. Refer to previous [release notes][] and [VERSION][] for additional details.
```

Comment:
`VERSION` is updated with `make docs-release TAG=v${MAJOR_VERSION}.${MINOR_VERSION}`. However, we do not generate versioned docs for RCs, only releases. This is documented in the `[Minor Release](#minor-release)` section.